### ISSUE TITLE
BdeClassifier docstring incorrectly states default loss is GaussianNLL

### ISSUE THREAD

**Issue by mbsuraj** [scikit-learn-contrib/bde#85]
This issue is for [JOSS Review #10653](https://github.com/openjournals/joss-reviews/issues/10653)

Inside bde.py, `BdeClassifier.__init__` docstring for the loss parameter incorrectly states GaussianNLL which is true for `BdeRegressor` but not for the classifier. 

The actual default for the classification (resolved in `FnnTrainer.default_loss`) is `CategoricalCrossEntropy`. 

**Suggested Fix**: Change the `BdeRegressor` docstring to 

```
loss : BaseLoss | None
    Custom training loss; defaults to :class:`bde.loss.CategoricalCrossEntropy`.

```

**Comment by mbsuraj** [comment]
Makes sense now. Thank you.
